fedimint-client/src/lib.rs

```diff
@@ -356,6 +374,10 @@ impl Client {
         self.inner.federation_meta.get(key).cloned()
     }
 
+    fn root_secret(&self) -> DerivableSecret {
```

Comment:
@elsirion Do we already have a way to pass module-specific secret to client module? Should it happen in the constructor (when the module instance is created?).

Reply:
I'm putting `root_secret` into `ClientInner`. I made that comment in a confusing way, as this line of code is about that, no module secrets.